x11-wm/2bwm/2bwm-0.3.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Between EAPI and inherit should be an empty line.

Reply:
Added empty line